client/src/components/ContentNavigator/index.ts

```diff
@@ -247,6 +250,74 @@ class ContentNavigator implements SubscriptionProvider {
           "workbench.actions.treeView.contentdataprovider.collapseAll",
         );
       }),
+      commands.registerCommand(
+        "SAS.convertSasnbToFlw",
+        async (resource: ContentItem) => {
+          const resourceUri = getUri(resource);
+          // Ensure that the command is only available for .sasnb files
+          if (resourceUri.path.endsWith(".sasnb")) {
+            // Open window to chose the name and location of the new .flw file
+            const name = await window.showInputBox({
+              prompt: "Enter the name for the new .flw file",
+              value: resource.name.replace(".sasnb", ".flw"),
+            });
+
+            if (!name) {
+              // User canceled the input box
+              return;
+            }
+
+            if (!name.endsWith(".flw")) {
+              window.showErrorMessage(Messages.InvalidFlwFileNameError);
+              return;
+            }
+
+            // check that name does not exist
+            const parent = await this.contentDataProvider.getParent(resource);
+            const children = await this.contentDataProvider.getChildren(parent);
+            if (children.some((child) => child.name === name)) {
+              window.showErrorMessage(Messages.FileAlreadyExistsError);
+              return;
+            }
+
+            try {
+              const contentUri: string =
```

Comment:
this variable name is misleading, it looks not a Uri

Reply:
resolved by b8c3d3cb4c704147184535b15badb5388f1d8fb5